libraries/AP_HAL_ChibiOS/hwdef/EAH743QF/README.md

```diff
@@ -0,0 +1,86 @@
+# Ewing Aerospace H743QF Flight Controller
+
+The Ewing Aerospace H743QF is a flight controller produced by [Ewing Aerospace](https://ewingaerospace.com/).
+
+## Features
+
+ - STM32H743 microcontroller
+ - ICM42688 IMU
+ - BMP390 barometer
+ - MAX7456 OSD
+ - 6 UARTs
+ - 6 PWM outputs
```

Comment:
could add info on BECs and GPIO controlled 10V supply, but its optional...just good marketing when this goes to a wiki page